roles/controller_casc_from_aap/tasks/inventory_from_aap_improved.yml

```diff
@@ -11,17 +11,17 @@
 - name: "Translate the inventories to a usefull structure"
   set_fact:
     current_inventories: "{{ (current_inventories | default([])) + [{
-      'has_inventory_sources': inventory_lookvar_item.has_inventory_sources,
-      'id': inventory_lookvar_item.id,
-      'name': inventory_lookvar_item.name,
-      'description': inventory_lookvar_item.description,
-      'organization': lookup('ansible.controller.controller_api', 'api/v2/organizations/'+(inventory_lookvar_item.organization|string), host=controller_hostname, username=controller_username, password=controller_password, verify_ssl=controller_validate_certs, return_all=true)['name'],
-      'host_filter': inventory_lookvar_item.host_filter,
-      'kind': inventory_lookvar_item.kind,
-      'variables': inventory_lookvar_item.variables,
-      'hosts': query('ansible.controller.controller_api', inventory_lookvar_item.related.hosts, host=controller_hostname, username=controller_username, password=controller_password, verify_ssl=controller_validate_certs, return_all=true) | json_query('[*].{name: name, description: description}'),
-      'groups': query('ansible.controller.controller_api', inventory_lookvar_item.related.groups, host=controller_hostname, username=controller_username, password=controller_password, verify_ssl=controller_validate_certs, return_all=true) | json_query('[*].{name: name, description: description, hosts_url: related.hosts}') | map('combine', {'inventory': inventory_lookvar_item.name}) | list
-    }] }}"
+        'has_inventory_sources': inventory_lookvar_item.has_inventory_sources,
```

Comment:
inconsistent indentation (4 spaces instead of 2)

Reply:
This was marked as an error identation by VScode and no errors when run ansible-lint